### DISCUSSION TITLE
[Feature Request] Allow overlapping and passing thumbs on the slider track

### DISCUSSION THREAD

**Discussion by Aryan-mor** [heroui-inc/heroui#4725] [Feature requests]
### Is your feature request related to a problem? Please describe.

I'm always frustrated when there are multiple thumbs on a single slider track, but they cannot overlap or pass each other.
This limitation restricts flexibility in use cases where values need to cross over.

Example:
https://react-range.pages.dev/?story=range--allow-overlap

### Describe the solution you'd like

I would like the ability for multiple thumbs on a single slider track to overlap and pass each other freely.
This would provide more flexibility and better UX when adjusting values that might intersect or move beyond one another.

### Describe alternatives you've considered

An alternative would be to allow partial overlap, where thumbs stop a short distance from each other but can still cross or interact in a restricted way.
However, full freedom of movement is preferred for smooth, unrestricted value adjustments.

**Comment by linear** [comment]
<p><a href="https://linear.app/nextui-inc/issue/ENG-1469/[feature-request]-allow-overlapping-and-passing-thumbs-on-the-slider">ENG-1469 [Feature Request] Allow overlapping and passing thumbs on the slider track</a></p>
